src/main/java/com/zendesk/maxwell/schema/MysqlSchemaStore.java

```diff
@@ -26,20 +28,22 @@ public MysqlSchemaStore(ConnectionPool maxwellConnectionPool,
 							BinlogPosition initialPosition,
 							CaseSensitivity caseSensitivity,
 							MaxwellFilter filter,
-							boolean replayMode) {
+							boolean readOnly) {
 		super(replicationConnectionPool, caseSensitivity, filter);
+		this.replicationConnectionPool = replicationConnectionPool;
```

Comment:
do you need to assign replicationConnectionPool and filter if you're calling super with them?